README.md

```diff
@@ -1 +1,37 @@
-# project_survey_form by Andrew and Akhtar
+# milestone-project/ phase-one 
+
+this is a project to learn the basics of how to use figma and flexbox
+
+### added html and css file to Github
+### created a linter to check for error in the HTML and CSS codes
+
+### created a file that should be ignored by Github
```

Comment:
- [ ] edit the README file to be up to date with the current repo. it will make you look more professional.

Reply:
- [ ] kindly re-phrase the comments politely, as suggestions instead of orders. For example:
    - This is a polite suggestion: _"I think it would be better to edit the README file to be up to date with the current repo. This will make your repository looks more professional."_
    - This reads as an order: _"edit the README file to be up to date with the current repo. it will make you look more professional."_